tests/end_to_end_integration/argo.yaml

```diff
@@ -35,6 +35,9 @@ spec:
               do_sat_adj: false
             gfdl_cloud_microphysics_nml:
               fast_sat_adj: false
+            gfs_physics_nml:
+              fhswr: 1800
+              fhlwr: 1800
```

Comment:
Previously the radiation timesteps were set to their default of 3600 seconds, which is longer than the segment length of this run (1800 seconds).  For reproducible restarts, the radiation timestep must evenly divide the segment length, so I have reduced the radiation timesteps here.

We might consider adding validation for this somewhere -- if the two are not compatible, then the model will crash in an uninformative way at the start of the second segment -- but I think that is outside the scope of this PR.